ui/src/app/store/machine/types.ts

```diff
@@ -13,9 +13,39 @@ export type Vlan = Model & {
   fabric_name: string;
 };
 
+export type Interface = Model & {
```

Comment:
I think it is a bit dangerous having a type called `Interface`, maybe `NetworkInterface` would be better!

Reply:
Yeah good point! Updating now